### PR TITLE
[RESTEASY-1422]

### PR DESCRIPTION
**Pull request by ronsigal**
Handle response headers with View entity in resteasy-html.

### CODE REVIEW

File: testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+@RunWith(Arquillian.class)
+@RunAsClient
+public class HeadersInViewResponseTest {
+
+    static ResteasyClient client;
+
+    @Deployment()
+    public static Archive<?> deploy() {
+        WebArchive war = TestUtil.prepareArchive(HeadersInViewResponseTest.class.getSimpleName());
+        war.addAsLibrary(getResteasyHtmlJar());
```
Comment (asoldano) [inline testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java]: Just asking, why not using the same same approach as in org.jboss.resteasy.utils.TestUtilSpring, basically reading the maven artifact from the Maven Repository?

Comment (ronsigal) [inline testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java]: Ah, well, I didn't know about it. Do you want me to use it?

Comment (asoldano) [inline testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java]: Not sure to be honest, I was wondering if you considered that and if there's a reason for preferring one of the two approaches.

Comment (ronsigal) [inline testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java]: Well, I can't think of a compelling reason one way or the other. MavenUtil is way more complicated and, presumably, more general. You could argue that using the maven repository is immune to Resteasy restructuring, but how often are we going to move resteasy-html?

Comment (asoldano) [inline testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/html/HeadersInViewResponseTest.java]: ok, let's keep this as is then, at least as long as no actual problem is found. Thanks.

**Pull request merged**